_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,1246 @@
+include the experimental IPC mining interface after it was first
+[introduced][news323 miningipc] last year.
+
+In June, StarkWare [demonstrated][news359 starkware] a modified Stratum v2
+client using STARK proofs to prove that a block's fees belong to a valid
+template without revealing the block's transactions. Two new Stratum V2-based
+mining pools also launched: [Hashpool][news346 hashpool], which represents
+mining shares as [ecash][topic ecash] tokens, and [DMND][news346 dmnd],
+which expanded from solo mining to pooled mining.
+
+</div>
+
+<div markdown="1" class="callout" id="releases">
+
+## Summary 2025: Major releases of popular infrastructure projects
+
+- [BDK wallet-1.0.0][] marked the first major release of this library, where the
+  original `bdk` crate was renamed to `bdk_wallet` with a stable API, and lower
+  layer modules were extracted into their own crates.
+
+- [LDK v0.1][] added support for both sides of the LSPS channel open negotiation
+  protocols, [BIP353][] Human Readable Names resolution, and reduced on-chain
+  fee costs when resolving multiple [HTLCs][topic htlc] for a single channel
+  force-closure.
+
+- [Core Lightning 25.02][] added support for [peer storage][topic peer storage],
+  off by default.
+
+- [Eclair v0.12.0][] added support for creating and managing [BOLT12
+  offers][topic offers], a new channel closing protocol that supports
+  [RBF][topic rbf], and support for storing small amounts of data for peers
+  through ([peer storage][topic peer storage]).
+
+- [BTCPay Server 2.1.0][] added several improvements for [RBF][topic rbf] and
+  [CPFP][topic cpfp] fee bumping, a better flow for multisig when all signers
+  are using BTCPay Server, and made some breaking changes for users of some
+  altcoins.
+
+- [Bitcoin Core 29.0][] replaced the UPnP feature (responsible in part for
+  several past security vulnerabilities) with a NAT-PMP option, improved
+  fetching of parents of orphan transactions for [package relay][topic package
+  relay], improved [timewarp][topic time warp] avoidance for miners, and
+  migrated the build system from `automake` to `cmake`.
+
+- [LND 0.19.0-beta][] added new RBF-based fee bumping for cooperative closes.
+
+- [Core Lightning 25.05][] introduced experimental [splicing][topic splicing]
+  support compatible with Eclair, and enabled peer storage by default.
+
+- [BTCPay Server 2.2.0][] added support for wallet policies and
+  [miniscript][topic miniscript].
+
+- [Core Lightning v25.09][] added support to the `xpay` command for paying
+  [BIP353][] addresses and [offers][topic offers].
+
+- [Eclair v0.13.0][] introduced an initial implementation of [simple taproot
+  channels][topic simple taproot channels], improvements to [splicing][topic
+  splicing] based on recent specification updates, and better BOLT12 support.
+
+- [Bitcoin Inquisition 29.1][] adds support for `OP_INTERNALKEY`, a type of
+  [covenant][topic covenants].
```

Comment:
I think its more accurate to say this opcode is part of some covenant proposals instead of a covenant itself? @Gustavojfe